### PR TITLE
Correct debug info + spelling/grammar/style

### PR DESCRIPTION
**Pull request by kerstin**
builds on #81; rewords confusing/misleading debug/log info at end of README + adds minor other fixes (spelling etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,23 +66,23 @@ GNU General Public License for more details](https://www.gnu.org/licenses/gpl-2.
 As I'm a fervent [Linux](https://www.kernel.org/) user, this version of the script has only been used and tested on Linux [Debian](https://www.debian.org/). I will not be able to maintain directly the [Windows](http://windows.microsoft.com) compatibility but I'm open to pull requests on this side, as we're here to share.
 
 #### Python version
-This version of TraktForVLC has been tested using Python 2.7.9 and Python 3.4.2 It should however be compatible with the following versions:
+This version of TraktForVLC has been tested using Python 2.7.9 and Python 3.4.2. It should, however, be compatible with the following versions:
 + Python 2.6
```
Comment (XaF) [inline README.md]: As you're in this file... can you please update the list of Python versions ? :)
+ Python 2.7
+ Python 3.3
+ Python 3.4
+ Python 3.5
+ Python 3.6
+ Python 3.7

(Thus, not 2.6 anymore, it was removed from the tests)

File: README.md [hunk 2]
```diff
@@ -66,23 +66,23 @@ GNU General Public License for more details](https://www.gnu.org/licenses/gpl-2.
...
 + Python 2.7
 + Python 3.3
 + Python 3.4
 
 #### VLC version
-This version of TraktForVLC has been tested using VLC 2.2.0 Weatherwax
+This version of TraktForVLC has been tested using VLC 2.2.0 Weatherwax.
```
Comment (XaF) [inline README.md]: The VLC version can be updated too: VLC 2.2.6 Umbrella

File: README.md [hunk 3]
```diff
@@ -246,6 +246,6 @@ and will automatically be started for subsequent system boots.
 ## Issues
 Please use the [GitHub integrated issue tracker](https://github.com/XaF/TraktForVLC/issues) for every problem you can encounter. Please **DO NOT** use my email for issues or walkthrough.
 
-When submitting an issue, please submit a TraktForVLC logfile showing the error. You can start TraktForVLC in debug mode (```--debug``` option) to obtain more thorough logs and with small timers (```--small-timers``` option) to speed up its internal loop. The resulting logfile (```TraktForVLC-DEBUG.log```) should then be available in your ```logs/``` directory.
+When submitting an issue, please submit a TraktForVLC logfile showing the errors you are running into. You can run TraktForVLC in debug mode (```--debug``` option) for more thorough logs when debugging manually. To save these contents into a separate logfile (```TraktForVLC-DEBUG.log```) in your ```logs/``` directory, use the ```--daemon``` option or ```--pidfile some_file.txt``` when running the script. You can additionally use the option ```--small-timers``` to speed up the script's internal loop.
```
Comment (XaF) [inline README.md]: Can you merge these changes with the other commit ?

File: README.md (hunk 1 above)
Comment (kerstin) [reply README.md]: Will do!

File: README.md (hunk 2 above)
Comment (kerstin) [reply README.md]: Will do!

File: README.md (hunk 3 above)
Comment (kerstin) [reply README.md]: Which one? (Sorry, this is showing as outdated, so didn't see the comment right away and I'm confused now, ha.)

Comment (XaF) [reply README.md]: Oh wait that's my bad, I did not see it was outdated and rewritten in the other commit :)

**Comment by kerstin** [comment]
Hi, because of the changed line numbers (with vs. without wrap) it's impossible to apply any of the content changes in this or the other PR without lots of extra work, so it will be easier if I reintroduce line breaks.

Please tell me at which character I should break. I have my rulers set to 78 for Python code and the max. recommendation (for console work exactly) is 80 iirc, but the current README on master goes quite a bit over that limit, and very inconsistently so, with some lines having > 90 characters even.

**Comment by XaF** [comment]
It's highly probable some of my changes in the README did not keep in sync with what should have been done, but I usually break at 80 (i.e. no line with 81 characters or more).

Also, please rebase your branch and squash all your commits in one (currently there is 4 of those)

**Comment by kerstin** [comment]
Added the line breaks back in. The branch was/is up-to-date with master, will squash the commits once I got your final OK with the current changes. (As far as I know GH PRs can also be squashed into one commit when accepting them though I don't have experience with that.)

**Comment by kerstin** [comment]
(In case this is something you wanna enable: [Configuring commit squashing for pull requests](https://help.github.com/articles/configuring-commit-squashing-for-pull-requests/))

**Comment by XaF** [comment]
I usually don't merge directly from GitHub. I don't know how the "squash and merge" process works and what will be the resulting commit message, so I prefer to avoid it. I usually merge by hand, so if you may squash them, I'll merge them after!

**Comment by kerstin** [comment]
Oh, interesting, I hadn't even considered merges not done on GH, so yeah, makes sense. In any case, I've now squashed them all down into one commit!

**Comment by kerstin** [comment]
I just had a look at the AppVeyor logs, looks like the build failing didn't have anything to do with the code? I haven't checked all of the Travis build instances, but what I saw was pointing to a problem with the test file...

**Comment by XaF** [comment]
As I said in one of the issues, because of everything else failing in TraktForVLC 1.x, I chose the path of giving more time on TraktForVLC 2.x to have a working TraktForVLC as fast as possible. I did not include your changes in the master as I did not take the time to fix the other problems lying around in 1.x Sorry for that!

TraktForVLC 2.x makes that PR obsolete - #88

**Pull request closed without merging**